string type
geometry_msgs/Pose pose
shape_msgs/Mesh geometry
